game/scripts/npc/npc_abilities_custom.txt

```diff
@@ -3,7 +3,13 @@
 #base "heroes/tusk.kv"
 #base "heroes/skywrath_mage.kv"
 
+#base "heroes/sniper.kv"
```

Comment:
I meant keep an open line between the #base lines and DOTAAbilities, so move this to line 5 and keep 6 open.